classes/04class/exercises/c04-rds02/christina-roperto/db-description.txt

```diff
@@ -0,0 +1,93 @@
+$ aws rds describe-db-instances
```

Comment:
can you include the parameter to specify which rds instance you want to describe?

Reply:
ok. will do. Thanks!